package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "discord.js": "14.11.0",
     "dotenv": "16.3.1",
+    "readdirp": "^3.6.0",
```

Comment:
Just a minor issue, can we use all exact version for dependencies as a convention?

Reply:
```suggestion
    "readdirp": "3.6.0",
```

Reply:
sure, no problem